sit_rezervo/notify/utils.py

```diff
@@ -11,3 +13,6 @@ def transfersh_direct_url(url: str):
     # prepend '/get' to the url path
     u = urlparse(url.strip())
     return u._replace(path=f"/get{u.path}").geturl()
+
+def activity_url(_class: Dict[str, Any]):
+    return f"<https://sit.biku.be/?activityId={_class['activityId']}|*{_class['name']}*>"
```

Comment:
The url host (`sit.biku.be`) should be a config value. Could be fixed later though.